src/utils/converter.ts

```diff
@@ -1,20 +1,32 @@
-import { utils } from 'near-api-js';
-import { Gas } from 'near-units';
-import { B_div_B, B_mul_N } from './math'
-
-
-const { formatNearAmount, parseNearAmount } = utils.format;
-
-const toSmall = (big: number | string, decimals: number): bigint => B_mul_N(BigInt("1" + "0".repeat(decimals)), big);
-const toLarge = (small: bigint, decimals: number): number => B_div_B(small, BigInt("1" + "0".repeat(decimals)))
+import { Big } from 'big.js';
+
+
+// config for Big.js behavior. see: https://mikemcl.github.io/big.js/
+Big.RM = 0;
+Big.DP = 40;
+Big.NE = -40;
+Big.PE = 40;
```

Comment:
Feel free to increase `Big.PE` to it's maximum `1e6`, because why not

Reply:
because performance